### PR TITLE
amend in api of replica set

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
change replica set to replicaset 
the kind: ReplicaSet ，I look the doc in https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
both use ReplicaSet objects
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #49619 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/openapi-spec/swagger.json [hunk 1]
```diff
@@ -57732,7 +57732,7 @@
     ]
    },
    "io.k8s.api.extensions.v1beta1.ReplicaSetCondition": {
-    "description": "ReplicaSetCondition describes the state of a replica set at a certain point.",
+    "description": "ReplicaSetCondition describes the state of a replicaset at a certain point.",
```
Comment (dixudx) [inline api/openapi-spec/swagger.json]: These json files are all auto-generated. Simply manually modify these don't work.

Comment (jianglingxia) [inline api/openapi-spec/swagger.json]: well,thanks for your review,but maybe not auto-generated,like pr #49592

**Comment by jianglingxia** [comment]
cc @ghodss 
/assign @dchen1107

**Comment by guangxuli** [comment]
There are a lot of places where used `replica set` words. Not sure this really needed.

**Pull request closed without merging**